templates/core/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,38 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Should this be in .gitignore and not in the PR?

Reply:
Since Terraform 0.14 dependency lock file is a recommendation to commit to git as per:
https://www.terraform.io/docs/language/dependency-lock.html

Reply:
Cool. Didn't know that.